lib/DDG/Spice/Thumbtack.pm

```diff
@@ -0,0 +1,42 @@
+package DDG::Spice::Thumbtack;
+# ABSTRACT: Returns service providers nearby
+
+use DDG::Spice;
+
+# Caching - https://duck.co/duckduckhack/spice_advanced_backend#caching-api-responses
+spice is_cached => 1;
```

Comment:
We actually want to disable this caching, because the same query, "electricians near me", won't always map to the same API call, as the call is dependent on the user's location